src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -169,6 +170,16 @@ public function getSymfonyMinorVersion()
         return $this->data['symfony_minor_version'];
     }
 
+    /**
+     * Returns if the current Symfony version is a Long-Term Support one.
+     *
+     * @return bool
```

Comment:
let's use a real return type instead

Reply:
by instead, you mean remove the comment?
or i keep it and also add the return type?

Reply:
As you've added the type hint in the method signature, you can remove the phpdoc as it is redundant now.